Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -161,7 +161,7 @@ void GcodeSuite::G33() {
     return;
   }
 
-  const bool towers_set           = parser.boolval('T', true),
+  const bool towers_set           = !parser.boolval('T'),
```

Comment:
Without the `T` parameter this will be true. With `T0` it will be true, and with `T1` it will be false. This probably changes the documentation.

Reply:
Good point!   @LVD-AC  can you get the documentation updated please?